src/webview/models.py

```diff
@@ -25,3 +33,138 @@ def create_profile(
 ) -> None:
     if created:
         Profile.objects.create(user=instance)
+
+
+class NotificationManager(models.Manager):
+    def create_for_user(
```

Comment:
it's a stylistic question, so feel free to entirely ignore it. I find that `_for_user` is a bit heavy and not necessarily required, as it is 1. conveyed by the argument (similarly, we could also push the logic and call the function `create_notification_for_user_with_title_and_message_and_is_read`), and 2. there's no `create` that could be confused with `create_for_user`. So I would just call it `create`.